frontend/src/components/7.Result/ResultPage.css

```diff
@@ -1,35 +1,37 @@
 .ResultTitle {
-  -webkit-text-stroke: 4px black;
   color: white;
-  font-family: Comic Sans MS;
-  margin: 1vw 1vw 0vw;
-  font-size: 50pt;
+  font-family:monospace;
```

Comment:
please remove this line. we shouldn't override the font family.

Reply:
Removed, thanks for letting me know that